pre-push-hook

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+
```

Comment:
This could go in the .github or scripts folders probably

Reply:
That's fair